Assignment2/ziader-ZiadJuneidi.ttl

```diff
@@ -0,0 +1,28 @@
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix : <http://example.org/#> .
+
+:Class01 a foaf:Class ;
+    rdfs:includes :Computer101 ;
+    rdfs:includes :Sensor029 ;
+.
+
+:Sensor029 a foaf:Class ;
+   rdfs:hasMeasurement :Measurement8401 ;
+.
+
+:Computer101 a foaf:Class ;
+   rdfs:hasOwner :User10A ;
+.
+
+:Measurement8401 a foaf:Class ;
+   rdfs:hasTemperature "29" ;
+   rdfs:atTime  "2010-06-12T12:00:12" ;
```

Comment:
There are many examples that you can consider and check in order to check how it should be written.